test/simplegraphs/generators/randgraphs.jl

```diff
@@ -309,4 +309,14 @@
     # testing domain errors
     @test_throws DomainError dorogovtsev_mendes(2)
     @test_throws DomainError dorogovtsev_mendes(-1)
+
+    # testing if returned graph is acyclic and valid SimpleGraph
+    rog = random_orientation_dag(SimpleGraph(5, 10))
```

Comment:
with the `rng` passed to it, you can test things further

Reply:
done